dali/pipeline/operators/fused/new_crop_mirror_normalize.h

```diff
@@ -95,18 +95,14 @@ class NewCropMirrorNormalize : public Operator<Backend>, protected CropAttr {
         DALI_FAIL("Not supported layout");
     }
 
-    DALI_ENFORCE(H >= crop_height_[data_idx] && W >= crop_width_[data_idx],
-      "Image dimensions for sample " + std::to_string(data_idx) + " ("
-      + std::to_string(H) + ", " + std::to_string(W) + ")"
-      + " are smaller than the cropping window" + " ("
-      + std::to_string(crop_height_[data_idx]) + ", "
-      + std::to_string(crop_width_[data_idx]) + ")");
-
-    auto crop_pos_y_x = CalculateCropYX(crop_y_norm_[data_idx], crop_x_norm_[data_idx],
-                                        crop_height_[data_idx], crop_width_[data_idx], H, W);
-
     auto crop_h = crop_height_[data_idx];
+    if (crop_h == 0)
```

Comment:
Use a boolean and populate it based on ArgumentDefined or TryGetValue.

Reply:
done